superlumic

```diff
@@ -112,10 +109,10 @@ if [ -d "/usr/local/superlumic/config" ]; then
 else
     if [ ! -z "$repo" ]; then
         setStatusMessage "Getting your config from your fork"
-        git clone -q $1 /usr/local/superlumic/config
+        git clone -q $repo /usr/local/superlumic/config
     else
         setStatusMessage "Getting the default config"
-        git clone -q https://github.com/superlumic/superlumic-config.git /usr/local/superlumic/config
+        git clone -q https://github.com/grozen/superlumic-config.git /usr/local/superlumic/config
```

Comment:
Thanks for your pull request grozen. I just started playing w/ this yesterday and I was having the same thoughts. One suggestion: change the clone repo back to superlumic so this pull request can be considered as-is.
Just my 2 cents.